### PR TITLE
feat: unauthorized component

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes

- Unauthorized components whose contents can be overwritten.
- As I noticed new pages with centered content are being introduced more and more, I created a dedicated container for it.
- This includes introducing a new tailwind CSS for min-height that considers the page header for computing the height itself.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-821 #done

### CODE REVIEW

File: packages/shared/src/components/Icon.tsx [hunk 1]
```diff
@@ -9,7 +9,8 @@ export type Size =
   | 'xlarge'
   | 'xxlarge'
   | 'xxxlarge'
-  | 'xxxxlarge';
+  | 'xxxxlarge'
+  | 'heading';
```
Comment (rebelchris) [inline packages/shared/src/components/Icon.tsx:13]: Maybe rather check with Design on naming as it needs to match with their system.
Not 100% sure if the sizing is still on point.

**Review by rebelchris — CHANGES_REQUESTED** [review]
Nice one!

One small remark on naming.

File: packages/shared/src/components/Icon.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/Icon.tsx:13]: Tomer just responded, he reverted the icon to be at `xxxlarge` size. Let me remove this new addition quickly.

File: packages/shared/src/components/errors/SquadLoading.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import classed from '../../lib/classed';
+
+const PlaceholderElement = classed('div', 'bg-theme-float');
+const RectangleElement = classed(PlaceholderElement, 'rounded-12');
+
+function SquadLoading(): ReactElement {
+  return (
+    <div className="flex overflow-hidden flex-col items-center pt-6 w-full max-h-page">
```
Comment (rebelchris) [inline packages/shared/src/components/errors/SquadLoading.tsx:9]: Something I only thought about now, did you make sure this also works neatly on mobile?

**Review by rebelchris — APPROVED** [review]
Nice just one non blocking question 🥳

File: packages/shared/src/components/errors/SquadLoading.tsx (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/components/errors/SquadLoading.tsx]: Good catch! I have now pushed a commit that displays like below:

mobile:
image

tablet:
image

laptop:
image

For the Unauthorized, it works very well on mobile devices.

**Pull request merged**